packages/components/inputs/select-input/src/select-input.tsx

```diff
@@ -46,7 +46,7 @@ type TCustomEvent = {
   persist: () => void;
 };
 
-type TSelectInputProps = {
+export type TSelectInputProps = {
```

Comment:
nit: In case of this components (and in several other cases) we're only exporting the `Props` type. This should be enough, but this limits users to referencing `TSelectInputProps` when typing the options object:

```tsx
import SelectInput, {
  type TSelectInputProps,
} from '@commercetools-uikit/select-input';

const options: TSelectInputProps['options'] = [
  { value: 'one', label: 'One' },
  { value: 'two', label: 'Two' },
];
const Example = (props: TSelectInputProps) => (
  <SelectInput
    options={options}
    // ...
  />
);
```

In case someone wished to be more explicit like:
```tsx
import /* ... */, {
  type TOption,
} from '@commercetools-uikit/select-input';

const options: TOption[] = [
  { value: 'one', label: 'One' },
  { value: 'two', label: 'Two' },
];
```
they have no chance of doing so. It wouldn't cost us much to export the additional types I think. Perhaps it's not needed 🤷‍♂️

Reply:
Thanks again Kacper 🙌🏾
Updated here: https://github.com/commercetools/ui-kit/pull/2429/commits/08aaffd670d8714e901916ef400811f18e97ea64